### ISSUE TITLE
That is not a valid source service worker file. Please try again with a file containing 'self.__WB_MANIFEST'.

### ISSUE THREAD

**Issue by AdeSupriyadi** [GoogleChrome/workbox#2438]
**Library Affected**:
*workbox-sw, workbox-cli*

**Browser & Platform**:
*Google Chrome latest*

**Issue or Feature Request Description**:
I follow tutorial from here https://developers.google.com/web/tools/workbox/guides/precache-files/cli

directory structure:
+**htdocs**
.........+-- **ibs-lite**
..................+--index.php
..................+--manifest.json
..................+--ibslite-sw.js
..................+--**workbox-v5.1.2**

**content of ibslite-sw.js**
![image](https://user-images.githubusercontent.com/14267724/78417584-ba6d9000-765d-11ea-9370-74f05bc3e979.png)

1. run **workbox wizard --injectManifest**
![image](https://user-images.githubusercontent.com/14267724/78417620-0e787480-765e-11ea-886b-b1be31c5f924.png)

I was add to ibslite-sw.js
`import {precacheAndRoute} from 'workbox-precaching';
precacheAndRoute(self.__WB_MANIFEST);`
but it not recognize.

any solution for it?

**Comment by AdeSupriyadi** [comment]
how to solve it?

**Comment by jeffposnick** [comment]
Does it work if you use a forward slash in the path, i.e. `ibs-lite/ibslite-sw.js`?

**Comment by srlhyo** [comment]
I also have the same problem. The documentation doesn't even specify what service worker I should create. Should the file have a conventional name? Also, do I need to add importScripts to the file?

**Comment by jeffposnick** [comment]
Apologies, I think this is due to https://github.com/GoogleChrome/workbox/issues/2447

While you're waiting for a fix, which should make it into Workbox v5.1.3, you should be able work around it by including something in a comment that gets the assertion to pass, like

```
// This commented line will allow validation to pass: precacheAndRoute([])

// This is the actual code:
precacheAndRoute(self.__WB_MANIFEST);
```

**Comment by jeffposnick** [comment]
(You can remove that extra line once the `workbox wizard` steps complete and you've got your configuration written out to disk.)

I'm closing this issue now in favor of tracking things in #2447.
